Using device: cuda
Autoencoder Model Summary
Traceback (most recent call last):
  File "/mnt/hpc/tmp/ralmeida/csci-3485-lab6/main.py", line 35, in <module>
    summary(autoencoder.encoder, (1, 28, 28))
  File "/mnt/hpc/tmp/ralmeida/csci-3485-lab6/env/lib/python3.12/site-packages/torchsummary/torchsummary.py", line 72, in summary
    model(*x)
  File "/mnt/hpc/tmp/ralmeida/csci-3485-lab6/env/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1553, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/hpc/tmp/ralmeida/csci-3485-lab6/env/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1562, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/hpc/tmp/ralmeida/csci-3485-lab6/env/lib/python3.12/site-packages/torch/nn/modules/container.py", line 219, in forward
    input = module(input)
            ^^^^^^^^^^^^^
  File "/mnt/hpc/tmp/ralmeida/csci-3485-lab6/env/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1553, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/hpc/tmp/ralmeida/csci-3485-lab6/env/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1603, in _call_impl
    result = forward_call(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/hpc/tmp/ralmeida/csci-3485-lab6/env/lib/python3.12/site-packages/torch/nn/modules/linear.py", line 117, in forward
    return F.linear(input, self.weight, self.bias)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument mat1 in method wrapper_CUDA_addmm)
[main 60a5c99] Finish lab6 job
 3 files changed, 34 insertions(+)
 create mode 100644 lab6.sh
 create mode 100644 slurm-3494.out
To github.com:rafaelolal/csci-3485-lab6.git
   078ee32..60a5c99  main -> main
